utils/src/main/java/com/cloud/utils/script/Script.java

```diff
@@ -242,13 +242,20 @@ public String execute(OutputInterpreter interpreter) {
                     if (_process.waitFor(_timeout, TimeUnit.MILLISECONDS)) {
                         //process completed successfully
                         if (_process.exitValue() == 0) {
-                            _logger.debug("Execution is successful.");
+                            _logger.debug("Execution is successfullish.");
```

Comment:
never knew the word `successfullish` before  :smiley:

Reply:
sorry, had to check if I was testing new code ;) will revert.